frontend/src/components/Forms/PartyRelationships/PermitteeOptions.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Field } from "redux-form";
+import { renderConfig } from "@/components/common/config";
+import { Form, Col, Row } from "antd";
+import { required } from "@/utils/Validate";
+import { createDropDownList } from "@/utils/helpers";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  mine: CustomPropTypes.mine,
+};
+
+const defaultProps = {
+  mine: {},
+};
+
+export const PermitteeOptions = (props) => {
+  const permitDropdown = createDropDownList(props.mine.mine_permit, "permit_no", "permit_guid");
+  return (
+    <Row gutter={16}>
+      <Col md={12} xs={24}>
+        <Form.Item>
+          <Field
+            id="related_guid"
+            name="related_guid"
+            label="Permit *"
+            placeholder="Select a Permit"
+            component={renderConfig.SELECT}
+            data={permitDropdown}
+            validate={[required]}
```

Comment:
This actually behaves as expected as is. It requires you select an option from the drop down and does not accept free-form input. A message is shown if nothing is selected. Perhaps parties behaves differently due to the way it's setting data and options rather than using a set of value/labels.